src/assignment/components/DuplicateAssignmentButton.tsx

```diff
@@ -35,7 +36,13 @@ const DuplicateAssignmentButton: FC<DuplicateAssignmentButtonProps & UserProps>
 			icon={IconName.copy}
 			{...props}
 			onClick={async (e) => {
-				const res = await duplicateAssignment(assignment, user as Avo.User.User);
+				if (!user?.profile?.id) {
+					ToastService.danger(
+						'Je moet ingelogd zijn om een opdracht te kunnen dupliceren'
+					);
```

Comment:
Extract translations

Reply:
fixed in https://github.com/viaacode/avo2-client/pull/1782